capture.py

```diff
@@ -91,6 +91,16 @@ def capture(camera=None,
     """
 
     camera = camera or "persp"
+    
+    # Fallback to default settings
+    if camera_options is None:
```

Comment:
Well, yes. Before the context managers copied the dictionaries. Now the capture method does it. Similarly (whether or not separating the context manager into individual ones again) the `apply` method could still hold this fallback and as such remove it from `capture()`.